docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/artifactory/index.md

```diff
@@ -0,0 +1,160 @@
+---
+sidebar_label: Artifactory
+---
+
+# Artifactory Webhook Receiver
+
+The Artifactory Webhook Receiver responds to `pushed` events originating from 
+Artifactory repositories by _refreshing_ all `Warehouse` resources subscribed to 
+those repositories.
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
+## Configuring the Receiver
+
+An Artifactory webhook receiver must reference a Kubernetes `Secret` resource 
+with a `secret-token` key in its data map. This
+[shared secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by
+Artifactory to sign requests and by the receiver to verify those signatures.
+
+:::note
+The following commands are suggested for generating and base64-encoding a
+complex secret:
+
+```shell
+secret_token=$(openssl rand -base64 48 | tr -d '=+/' | head -c 32)
+echo "Secret token: $secret_token"
+echo "Encoded secret token: $(echo -n $secret_token | base64)"
+```
+
+:::
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Project
+metadata:
+  name: kargo-demo
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: private-repo-creds
+  namespace: kargo-demo
+  labels:
+    kargo.akuity.io/cred-type: image
+data:
+  repoURL: <jfrog-instance>.jfrog.io/<repo-key>/<image-name>
+  username: <jfrog username>
+  password: <jfrog docker login password>
```

Comment:
Thanks for splitting this out from the other Secret. There's not much need to include these repo creds at all though since the doc isn't walking anyone through setting up a Warehouse (and doesn't need to). Since this is reference material we can afford to zero in on the artifactory --> receiver communication and not worry about Warehouse configuration. There are other docs that cover that well.